src/plugins/CreateExecution/CreateExecution.js

```diff
@@ -187,10 +185,12 @@ define([
 
         if (snapshot) {
             return Q.all(nodes.map(node => {
-                if (this.isMetaTypeOf(node, this.META.Operation)) {
-                    return this.snapshotNode(node, dst);
-                } else if (this.isMetaTypeOf(node, this.META.Transporter)) {
+                if (this.isLocalOperation(node) ||
```

Comment:
This is the most important line. Basically, I am simply copying the local operations and transporters (connections) rather than just the transporters.

Most of the rest of the code is refactoring to allow the CreateExecution plugin know about local operations